content/updates/2020-07/consistency-in-structure.md

```diff
@@ -0,0 +1,15 @@
+# Avoid spelling mistakes in your product data
+::: meta-data type="Improvement" features="Data Quality Insights" available="early July" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
+:::
+
+We keep on adding new features to help you improve your data quality. To garantee consistency on your customer-facing product data, we introduced spell-check in category names, attribute and options labels.
+You can now check spelling mistakes and correct them in your categories and attributes data quality in the `Settings`. 
+This improvement is also available in the `Product Edit Form`. New recommendations are listed should we find errors. The better the content, the higher the consistency grade :wink:
```

Comment:
```suggestion
This improvement is also available in the product form. New recommendations are listed should we find errors. The better the content, the higher the consistency grade! :wink:
```

Reply:
As we talk about the consistency grade, I feel like a "What is the consistency grade?" in the More block would be more than welcomed. :)